README.md

```diff
@@ -1,15 +1,10 @@
 # Portfolio
```

Comment:
- [ ] [Optional] Kindly add a description to the README and remove the empty sections (from line ```17``` to ```28```), this will make your documentation more professional and clean